cmd/agent/subcommands/run/command.go

```diff
@@ -474,6 +473,7 @@ func getSharedFxOption() fx.Option {
 		networkpath.Bundle(),
 		remoteagentregistryfx.Module(),
 		haagentfx.Module(),
+		metricscompressorfx.Module(),
 	)
```

Comment:
What's the reason we don't need the `logscompression.Module()` here? (while for instance we need it in the JMX command. FWIW it seems we need it in all other commands)

Reply:
It is being provided by `process.Bundle()`. On reflection, I don't think it should be so I will move it out.